qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -289,3 +321,114 @@ def default_values(self) -> List[DefaultCalValue]:
                     defaults.append(DefaultCalValue(value, param.name, tuple(), name))
 
         return defaults
+
+
+class EchoedCrossResonance(BasisGateLibrary):
+    """A fixed-frequency transmon gate library with echoed cross-resonance gates.
+
+    This library extends the FixedFrequencyTransmon library by adding support for
+    cross-resonance gates.
+    """
+
+    __default_values__ = {
+        "cr_duration": 640,
+        "cr_width": 384,
+        "cr_σ": 64,
+        "cr_amp": 0.5,
+        "rot_amp": 0.0,
+    }
+
+    def __init__(
+        self,
+        single_qubit_library: BasisGateLibrary,
+        rotary: bool = True,
```

Comment:
Can this be renamed to something like `target_pulse` to cover both rotary and active cancellation? Perhaps this is too broad?